composer.json

```diff
@@ -20,23 +20,25 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
     },
     "extra": {
         "laminas": {
             "module": "Laminas\\ApiTools\\Documentation\\ApiBlueprint"
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas-api-tools/api-tools-api-problem": "^1.2.1",
         "laminas-api-tools/api-tools-content-negotiation": "^1.2.1",
         "laminas-api-tools/api-tools-documentation": "^1.2",
         "laminas/laminas-eventmanager": "^3.2",
         "laminas/laminas-mvc": "^2.7.15 || ^3.0.4",
         "laminas/laminas-servicemanager": "^2.7.6 || ^3.1",
-        "laminas/laminas-view": "^2.8.1",
-        "laminas/laminas-zendframework-bridge": "^1.0"
```

Comment:
To remove `laminas/laminas-zendframework-bridge`, we first need to add a conflict with `zfcampus/zf-apigility-documentation-apiblueprint`

Reply:
I'm focused now on PHP 8.1 so reverted